### PR TITLE
CLOUD-443 Big data test implementation

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| recreate | passed |
| scaling | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| storage | failed |
| demand-backup | passed |
| operator-self-healing | passed |
| scheduled-backup | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| init-deploy | passed |
| recreate | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| storage | passed |
| demand-backup | passed |
| operator-self-healing | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | failed |
| scheduled-backup | passed |

File: e2e-tests/big-data/run [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    kubectl_bin apply -f "${test_dir}/conf/backup.yml"
+    sleep 10
+
+    kubectl_bin apply -f "${test_dir}/conf/restore-backup.yml"
+
+    wait_backup_restore "restore1"
+
+    wait_for_running "$cluster-proxysql" 1
+    wait_for_running "$cluster-pxc" 3
+
```
Comment (delgod) [inline e2e-tests/big-data/run]: could you please add SST test? like delete PVC and Pod

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | failed |
| init-deploy | passed |
| recreate | passed |
| scaling | passed |
| storage | passed |
| scaling-proxysql | passed |
| demand-backup | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | failed |
| scheduled-backup | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| self-healing | failed |
| recreate | failed |
| scaling | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| recreate | passed |
| init-deploy | passed |
| big-data | failed |
| scaling | passed |
| storage | failed |
| scaling-proxysql | passed |
| operator-self-healing | passed |
| demand-backup | passed |
| scheduled-backup | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| init-deploy | passed |
| storage | failed |
| big-data | failed |
| self-healing | passed |
| operator-self-healing | passed |
| scaling | passed |
| scaling-proxysql | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| operator-self-healing | passed |
| big-data | failed |
| recreate | passed |
| demand-backup | passed |
| scaling | passed |
| scheduled-backup | passed |
| scaling-proxysql | passed |
| init-deploy | passed |
| storage | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| operator-self-healing | passed |
| scaling | passed |
| scaling-proxysql | passed |
| big-data | failed |
| init-deploy | passed |
| storage | passed |
| limits | passed |
| recreate | passed |
| monitoring | passed |
| monitoring-2-0 | failed |
| demand-backup | passed |
| scheduled-backup | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| operator-self-healing | passed |
| scaling | passed |
| scaling-proxysql | passed |
| recreate | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| init-deploy | passed |
| storage | passed |
| limits | passed |
| big-data | failed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| self-healing | failed |
| recreate | failed |
| scaling | passed |
| big-data | failed |
| scaling-proxysql | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| init-deploy | passed |
| scaling | passed |
| big-data | failed |
| recreate | passed |
| scaling-proxysql | passed |
| storage | passed |
| demand-backup | passed |
| operator-self-healing | passed |
| limits | passed |
| monitoring | passed |
| auto-tuning | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| scheduled-backup | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| big-data | failed |
| scaling | passed |
| recreate | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| storage | failed |
| operator-self-healing | passed |
| demand-backup | passed |
| auto-tuning | passed |
| scheduled-backup | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| big-data | passed |
| recreate | passed |
| self-healing | failed |
| storage | passed |
| scaling-proxysql | passed |
| demand-backup | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| scheduled-backup | failed |
| affinity | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| recreate | passed |
| big-data | passed |
| demand-backup | passed |
| scaling | passed |
| scheduled-backup | failed |
| scaling-proxysql | passed |
| init-deploy | passed |
| storage | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| operator-self-healing | passed |
| auto-tuning | passed |

**Pull request merged**